filament/backend/src/opengl/OpenGLProgram.h

```diff
@@ -74,10 +74,10 @@ class OpenGLProgram : public backend::HwProgram {
     struct BlockInfo {
         uint8_t binding : 3;    // binding (i.e.: index in mSamplerBindings)
         uint8_t unused  : 1;    // padding / available
-        uint8_t count   : 4;    // number of TMUs actually used minus 1
+        uint8_t count   : 5;    // number of TMUs actually used minus 1
```

Comment:
you need to remove the `unused` field here. Otherwise we get 9 bits, which doesn't fit on a byte anymore.

Reply:
That's good point. I don't know why I can't find `unused`. It's removed.